packages/opentelemetry-core/src/metrics/NoopMeter.ts

```diff
@@ -137,3 +143,5 @@ export const NOOP_MEASURE_HANDLE = new NoopMeasureHandle();
 export const NOOP_MEASURE_METRIC = new NoopMetric<MeasureHandle>(
   NOOP_MEASURE_HANDLE
 );
+
+export const NOOP_METER = {} as LabelSet;
```

Comment:
Maybe `s/NOOP_METER/NOOP_LABEL_SET`? Looks more appropriate for me. WDYT?

Reply:
Agree, update to `NOOP_LABEL_SET`